Python/sysmodule.c

```diff
@@ -1941,6 +1941,37 @@ sys__clear_type_cache_impl(PyObject *module)
     Py_RETURN_NONE;
 }
 
+#ifdef Py_DEBUG
+/*[clinic input]
+sys._get_type_version_tag
```

Comment:
I would prefer to put this function in the _testcapi module, since it seems to only be useful for a single unit test.